src/components/Book.js

```diff
@@ -15,14 +16,43 @@ function Book({
     });
     dispatch(removeBook(id));
   };
+  const percentage = 67;
+
   return (
-    <>
-      <li className="book-list">
-        <p>{title}</p>
-        <p>{category}</p>
-        <button type="button" onClick={remove}>Remove</button>
-      </li>
-    </>
+    <div className="book-cards">
+      <div className="book-list">
+        <p className="book-category">{category}</p>
+        <p className="book-title">{title}</p>
+        <button type="button" className="btn-modify">Comment</button>
+        <button type="button" onClick={remove} className="btn-modify">Remove</button>
+        <button type="button" className="btn-modify">Edit</button>
```

Comment:
- [ ] In order to stick the design as much as possible as the previous reviewer requested, kindly put bars between `Comment Remove Edit` buttons and also make `category names` bolder below is the original design to compare👇
![Screenshot (142)_LI](https://user-images.githubusercontent.com/67087664/150634015-afe5e153-fe66-4a01-be3a-8daddab3064a.jpg)

- Your design 👇 
![Screenshot (145)](https://user-images.githubusercontent.com/67087664/150634076-22e5dae0-208c-4479-a07d-5d4622f48769.png)